src/main/java/com/connexta/search/index/IonResourceLoader.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.search.index;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import lombok.AllArgsConstructor;
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.ResourceLoader;
+
+@AllArgsConstructor
+public class IonResourceLoader {
+  private final ResourceLoader resourceLoader;
+
+  // TODO: Not intended to be permanent API
+  public String getAsString(String resourceUrl) throws IOException {
+    return IOUtils.toString(get(resourceUrl), StandardCharsets.UTF_8);
+  }
+
+  public InputStream get(String resourceUrl) throws IOException {
+    return resourceLoader.getResource(resourceUrl).getInputStream();
```

Comment:
✏️ I'm not sure this will work for our services. This will be a URL like `http://blah:8080/dataset/abc123/irm`. That endpoint will require an accept version in the request headers.

Reply:
Actually, since we do not define the retrieve endpoint in the API we won't need the accept version. But eventually this index service will need to be aware of the store version it is communicating with.

Reply:
We currently don't use accept-versions for our retrieve endpoints.

Reply:
@ahoffer, have you tested the ResourceLoader with out retrieve endpoints?

Reply:
Why will search need to be aware of the version in the future?

Reply:
Will there be any problems with the resource loader when we switch to https?

Reply:
Do we get the resource from the URL every time we need an input stream?